accord-core/src/main/java/accord/primitives/Timestamp.java

```diff
@@ -365,6 +365,11 @@ static <T extends Timestamp> T merge(Timestamp a, Timestamp b, Constructor<T> co
 
     @Override
     public String toString()
+    {
+        return toStandardString();
+    }
+
+    public String toStandardString()
```

Comment:
so, when we serializer in some cases we only know `Timestamp` but we have many types... and almost all override `toString` so the parser of string to `Timestamp` breaks...  By making this method it makes sure we can always do the following property

```
new Timestamp(this).equals(Timestamp.parse(this.toStandardString()))
```

Reply:
this is not a core change, it was just found in testing....